### PR TITLE
Fix issues identified during 2.0-beta-1 testing

### PR DESCRIPTION
**Pull request by daniel-beck**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/matrixauth/AuthorizationMatrixNodeProperty.java [hunk 1]
```diff
@@ -172,6 +172,12 @@ public boolean isApplicable(Class<? extends Node> folder) {
             return isApplicable();
         }
 
+        @Nonnull
+        @Override
+        public String getDisplayName() {
+            return Messages.AuthorizationMatrixNodeProperty_DisplayName();
```
Comment (daniel-beck) [inline src/main/java/org/jenkinsci/plugins/matrixauth/AuthorizationMatrixNodeProperty.java]: Default method didn't work (Descriptor already implements it), and forgot i18n.

File: src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly [hunk 2]
```diff
@@ -32,14 +32,14 @@ THE SOFTWARE.
           <j:when test="${attrs.sid == 'authenticated'}">
             <td class="left-most">
               <span title="authenticated">
-                <img src="${imagesURL}/16x16/user.gif" style="margin-right:0.2em"/>${%Authenticated Users}
+                <img src="${imagesURL}/16x16/user.png" style="margin-right:0.2em"/>${%Authenticated Users}
```
Comment (daniel-beck) [inline src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly]: Added opacity.

File: src/main/resources/org/jenkinsci/plugins/matrixauth/inheritance/InheritParentStrategy/config.properties [hunk 3]
```diff
@@ -1,2 +1,2 @@
-blurb = This item will inherit its parent item's permissions (in addition to any permissions granted here).\
+blurb = This item will inherit its parent item's permissions (in addition to any permissions granted here). \
```
Comment (daniel-beck) [inline src/main/resources/org/jenkinsci/plugins/matrixauth/inheritance/InheritParentStrategy/config.properties]: Missing space.

File: src/main/resources/org/jenkinsci/plugins/matrixauth/inheritance/NonInheritingStrategy/config.properties [hunk 4]
```diff
@@ -1,3 +1,4 @@
 blurb = This object will <strong>not</strong> inherit the <a href="{0}/configureSecurity">global security security settings</a>, or any permissions from its ancestors. \
+  Only permissions explicitly enabled here will be granted. \
```
Comment (daniel-beck) [inline src/main/resources/org/jenkinsci/plugins/matrixauth/inheritance/NonInheritingStrategy/config.properties]: Let's be clear here.

File: src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly (hunk 2 above)
Comment (oleg-nenashev) [reply src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly]: ty

**Pull request merged**